zerver/webhooks/stripe/view.py

```diff
@@ -53,6 +55,14 @@ def api_stripe_webhook(
     payload: Dict[str, Any] = REQ(argument_type="body"),
     stream: str = REQ(default="test"),
 ) -> HttpResponse:
+    sig_header = request.META["HTTP_STRIPE_SIGNATURE"]
+    try:
+        stripe.Webhook.construct_event(request.body, sig_header, get_secret("stripe_api_endpoint"))
+    except ValueError:
+        return HttpResponse(status=400)
+    except stripe.error.SignatureVerificationError:
+        return HttpResponse(status=400)
```

Comment:
We should add automated tests for this. And the docs will need to be updated as well.